src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -779,33 +779,24 @@ public Controller getController()
      * configured in the arena selector of this strategy.
      *
      * @param sstables a collection of the sstables to be assigned to arenas
-     * @param compactionFilter a filter to exclude CompactionSSTables,
+     * @param compactionFilter a bifilter (sstable, isCompacting) to exclude CompactionSSTables,
      *                         e.g., {@link CompactionSSTable#isSuitableForCompaction()}
```

Comment:
I think the example mentioned here is not accurate anymore, as `CompactionSSTable#isSuitableForCompaction()` is a simple `Predicate`.